SafeSurroundings/SafeSurroundings.Data/Services/InMemoryMeetUpTable.cs

```diff
@@ -18,7 +18,14 @@ public InMemoryMeetUpTable()
 
         public void Add(MeetUp newMeetup)
         {
+            try
+            {
             newMeetup.ID = meetUps.Max(m => m.ID )+ 1;
+            }
+            catch
+            {
+                newMeetup.ID = Convert.ToInt32(DateTime.Now.ToString("mmss"));
```

Comment:
Replacing with a random number instead of a timestamp for testing. The number generated needs to be within the integer range and could also fall under the same value on different days.